app/src/main/java/org/mozilla/fenix/tabstray/inactivetabs/InactiveTabs.kt

```diff
@@ -0,0 +1,280 @@
+ *
+ * @param expanded Whether the section is expanded.
+ * @param onClick Called when the user clicks on the header.
+ * @param onDeleteAllClick Called when the user clicks on the delete all button.
+ */
+@Composable
+private fun InactiveTabsHeader(
+    expanded: Boolean,
+    onClick: () -> Unit,
+    onDeleteAllClick: () -> Unit,
+) {
+    ExpandableListHeader(
+        headerText = stringResource(R.string.inactive_tabs_title),
+        expanded = expanded,
+        expandActionContentDescription = stringResource(R.string.inactive_tabs_expand),
+        collapseActionContentDescription = stringResource(R.string.inactive_tabs_collapse_group),
+        onClick = onClick,
+    ) {
+        IconButton(
+            onClick = onDeleteAllClick,
+            modifier = Modifier.padding(horizontal = 8.dp),
+        ) {
+            Icon(
+                painter = painterResource(R.drawable.ic_delete),
+                contentDescription = stringResource(R.string.inactive_tabs_delete_all),
+                modifier = Modifier.size(20.dp),
+                tint = FirefoxTheme.colors.iconPrimary,
+            )
+        }
+    }
+}
+
+/**
+ * Inactive Tabs auto close dialog.
+ *
+ * @param onDismissClick Called when the user clicks on the auto close dialog's dismiss button.
+ * @param onEnableAutoCloseClick Called when the user clicks on the auto close dialog's enable button.
+ */
+@Composable
+private fun InactiveTabsAutoClosePrompt(
+    onDismissClick: () -> Unit,
+    onEnableAutoCloseClick: () -> Unit,
+) {
+    Card(
+        modifier = Modifier.padding(horizontal = 16.dp, vertical = 12.dp),
+        shape = ROUNDED_CORNER_SHAPE,
+        backgroundColor = FirefoxTheme.colors.layer2,
+        border = BorderStroke(
+            width = 1.dp,
+            color = FirefoxTheme.colors.borderPrimary,
+        ),
+    ) {
+        Column(
+            modifier = Modifier.padding(start = 12.dp),
+            horizontalAlignment = Alignment.End,
+        ) {
+            Row(
+                verticalAlignment = Alignment.CenterVertically,
+            ) {
+                PrimaryText(
+                    text = stringResource(R.string.tab_tray_inactive_auto_close_title),
+                    modifier = Modifier.weight(1f),
+                    fontSize = 14.sp,
+                    fontFamily = FontFamily(Font(R.font.metropolis_semibold)),
+                )
+
+                IconButton(
+                    onClick = onDismissClick
+                ) {
+                    Icon(
+                        painter = painterResource(R.drawable.mozac_ic_close),
+                        contentDescription =
+                        stringResource(R.string.tab_tray_inactive_auto_close_button_content_description),
+                        modifier = Modifier.size(20.dp),
+                        tint = FirefoxTheme.colors.iconPrimary,
+                    )
+                }
+            }
+
+            SecondaryText(
```

Comment:
I intentionally haven't used any updated specs for the inactive tabs-specific content. This has honestly been in review so long that I don't remember why I used `SecondaryText`, but it could have been to mirror collections. I'm happy to switch it to `Primary` though